demo/demo.html

```diff
@@ -4,14 +4,14 @@
 
         <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/3.10.1/lodash.js"></script>
         <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jszip/2.5.0/jszip.min.js"></script>
-        <script type="text/javascript" src="../dist/excel-builder.dist.js"></script>
+        <script type="text/javascript" src="../dist/index.js"></script>

```

Comment:
Sorry, direct loading here would not work. Because ncc only builds commonjs dist file, not umd.

Probably can remove the whole demo/ folder from this repo.